app.js

```diff
@@ -1,19 +1,19 @@
 'use strict';
 var express = require('express'),
-	fs = require('fs'),
-	JSON5 = require('json5'),
+    bodyParser = require('body-parser'),
 	app = express(),
-	jsonData;
+    messages = [];
 
-var questions = [];
+app.use(express.static('public'));
+app.use(bodyParser.urlencoded({ extended: false }));
 
-fs.readFile('./public/freebase_countries.json', 'utf8', function(err, data){
-    jsonData = JSON5.parse(data);
-    getDataFromJson(jsonData, questions);
-    console.log(questions);
+app.post('/messages', function(req, res) {
+	  messages.push(req.body.message);
```

Comment:
you need to send HTTP response. Always. it can be just empty response. `res.send(201)`